store.html

```diff
@@ -0,0 +1,166 @@
+                <input type="text" class="name" name="lname" required></div>
+            </div>
+                    
+            <label>Address*</label>
+            <input class="address" type="text" required><br>
+                    
+            <label>Address Line 2*</label>
+            <input class="address" type="text"><br>
+                    
+<!--note flex-->
+<div class="parent--form ">
+    <div class="div--name"> 
+        <label for="city">City*</label>
+        
+                    <select id="cities" name="city" form="city_form">
+                      <option value="disabled">Select option</option>
+                       <option value="london">London</option>
+                       <option value="milton keynes">Milton Keynes</option>
+                       <option value="luton">Luton</option>
+                       <option value="leeds">Leeds</option>
+                       <option value="manchester">Manchester</option>
+                       <option value="plymouth">Plymouth</option>
+                       <option value="bristol">Bristol</option>
+                       <option value="liverpool">Liverpool</option>
+                       <option value="glasgow">Glasgow</option>
+                     </select>
+    </div>
+                
+              
+                <div class="div--name">
+                    <label for="postcode">Post Code*</label>
+                     <input type="text" id="postcode" name="postcode">
+                </div>
+</div>           
+                    
+                    
+ <p class="pform">Select a colour</p>   
+            <div class="colour">
+                    
+
+                    <div class="radio--content">
+                    <input type="radio" name="karma-orange" id="karma--orange" value="karma--orange">
+                    <label for="html">Karma Orange</label></div>
+                    
+                    <div class="radio--content">
+                    <input type="radio" id="Space-Grey" name="Space-Grey" >
+                    <label for="html">Space Grey</label></div>
+                  </div><br>
+                  
+                
+                  <div>
+                    <input type="checkbox" id="checkbox" name="checkedbox">
+                    <label for="checkbox">By placing your order, you agree to Karma's<a class="terms"href="#">Terms and Conditions</a></label>
+                  </div>
+              
+                <button class="btn btn2" type="submit" value="ikaneng.khoza@gmail.com">Place Order</button>
+     
+            
+ </form>
+              
+     
+     <div class="img--div">
+        <img src="img/Girl-working-at-desk.png">
+    <div>          
+              
+    </section> 
+</main>
+
+
+<footer class="footer--wrap">
+    <div><h2>Join us on</h2></div> 
+<div class="social">
+   
+
+<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 50 50">
```

Comment:
Instead of using `svg` element, I would put this into an image and name it twitter.svg and attach this as an image instead. This way, it doesn't clutter your html